internal/controller/builder/builder.go

```diff
@@ -844,20 +843,6 @@ func BuildEnvConfig(cluster *appsv1alpha1.Cluster, component *component.Synthesi
 		}
 	}
 
-	// build common env, but not for statelsss workload
-	if component.WorkloadType != appsv1alpha1.Stateless {
-		commonEnv := buildWorkloadCommonEnv(cluster, component)
```

Comment:
why not build commont envs?

Reply:
built them in rsm as common envs are component scope staffs.